sdc/rabbit/publisher.py

```diff
@@ -35,6 +36,7 @@ def __init__(self, urls, queue, **kwargs):
         self._arguments = kwargs
         self._connection = None
         self._channel = None
+        self.confirm_delivery = confirm_delivery
```

Comment:
`self._confirm_delivery` - should be a private attribute, as once a channel is instantiated in confirm mode it cannot be changed, so a user shouldn't change this attribute after instantiating their object.